packages/debugger/style/kernelSources.css

```diff
@@ -34,3 +34,19 @@
 .jp-DebuggerKernelSource-filterBox-hidden {
   display: none;
 }
+
+.jp-DebuggerKernelSource-source {
+  display: flex;
+  align-items: center;
+  padding: 4px;
+  cursor: pointer;
+}
+
+.jp-DebuggerKernelSource-source:hover {
+  background-color: var(--jp-layout-color2);
+}
+
+.jp-DebuggerKernelSource-source svg {
```

Comment:
Please replace the bare use of `svg` with a selector or `>`

Reply:
Thanks @krassowski, I thought the selector checked the first element before the others.

Fixed in https://github.com/jupyterlab/jupyterlab/pull/15021/commits/31d2e47b92a0b72a6a440c1cd8c9c232ac43d8e7